scout/constants/disease_parsing.py

```diff
@@ -1,7 +1,7 @@
 import re
 
-MIMNR_PATTERN = re.compile("[0-9]{6,6}")
-ENTRY_PATTERN = re.compile("\([1,2,3,4]\)")
+MIMNR_PATTERN = re.compile(r"[0-9]{6,6}")
+ENTRY_PATTERN = re.compile(r"\([1-4]\)")
```

Comment:
Hm, this looks lite it would have broken to me, but maybe it was ok enough. Or maybe the logic just never kicked in - let's see!

Reply:
Ah the backslashes needed to be escaped?

Reply:
Yes, I think so - the `\(\)` would be converted to `()` before compiling the re, and the intent is to match literal parenthesis characters in the file not to group number matches.